### PR TITLE
[docs] Add HPA and VPA to limitations

### PR DESCRIPTION
**Pull request by aravindhp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,10 @@ workloads.
 ### Storage
 At this time, only in-tree storage is supported in all cloud providers.
 
+### Pod Autoscaling
+[Horizontal](https://docs.openshift.com/container-platform/4.11/nodes/pods/nodes-pods-autoscaling.html) and
+[Vertical](https://docs.openshift.com/container-platform/4.11/nodes/pods/nodes-pods-vertical-autoscaler.html) Pod
+autscaling support are not available for Windows workloads.
```
Comment (mburke5678) [inline README.md:240]: s/autscaling/autoscaling

**Comment by mburke5678** [comment]
@aravindhp One small typo. Otherwise LGTM.

File: README.md (hunk 1 above)
Comment (aravindhp) [reply README.md]: Thanks for catching that. Fixed.

**Comment by sebsoto** [comment]
/approve
besides from above comment

**Pull request merged**